easyrest/views/restaurant_controler.py

```diff
@@ -156,11 +170,100 @@ def get_my_restaurant_controler(request):
                 },
             ]
     """
-    owner = request.headers.get('Authorization')
-    # TODO: take owner_id from table users after Max pullrequst
-    # TODO: token = request.headers.get('Authorization')
-    # TODO: own = request.dbsession.query(User).filter_by(token=token).first()
-    # TODO: owner = own[0]['owner_id']
-    restaurants = asign_tags(request.dbsession.query(Restaurant).filter_by(owner_id=owner).all())
+    restaurants = asign_tags(request.token.user.restaurants)
     body = wrap(restaurants)
+
     return body
+
+
+@view_config(
+    route_name='user_restaurants',
+    request_method="POST",
+    renderer='json'
+)
+@restrict_access(user_types=['Client'])
+def create_user_restaurant(request):
+    """
+    POST request controller. Create new restaurant in database and return created item
+    """
+    rest_data = request.json_body
+
+    name, description, phone, address, tags = rest_data["name"], rest_data[
+        "description"], rest_data["phone"], rest_data["address"], rest_data["tags"]
+
+    if not name or not address:
+        msg = "Fill all required fields"
+        return Response(status=422, json_body=wrap([], success=False, error=msg))
+
+    tag_models = [request.dbsession.query(
+        Tag).filter_by(name=tag["name"]).first() for tag in tags]
+
+    rest = Restaurant(name=name, description=description,
+                      phone=phone, address_id=address)
+    rest.tag = tag_models
+    rest.user = request.token.user
+    try:
+        request.dbsession.add(rest)
+        request.dbsession.flush()
+        rest_with_tags = asign_tags([rest])
+        return Response(status=201, json_body=wrap(rest_with_tags, action="Restaurant was successfully created"))
```

Comment:
Instead of returning response you'd better set `request.response.status_code = 201` and return `wrap(...)`. As Oleksandr did in his PR #9